pilot/pkg/config/clusterregistry/conversion.go

```diff
@@ -49,9 +47,10 @@ const (
 	// The cluster's platform: Kubernetes, Consul, Eureka, CloudFoundry
 	ClusterPlatform = "config.istio.io/platform"
 
-	// The cluster's access configuration file
+	// The cluster's access configuration stored in k8s Secret object
 	// E.g., on kubenetes, this file can be usually copied from .kube/config
-	ClusterAccessConfigFile = "config.istio.io/accessConfigFile"
+	ClusterAccessConfigSecret          = "config.istio.io/accessConfigSecret"
```

Comment:
Do we really need the namespace ? The config file should have a ref to the secret.

Reply:
@costinm I think we do due to security reason expressed above, but again it is optional.

Reply:
If you really need it and it's optional - ok, but we should have tests and docs without using it ( and only mention it for advanced setups ).